contributions/essay/aronber-yannik/Readme.md

```diff
@@ -0,0 +1,14 @@
+# The rust language build pipeline
+
+In this essay we would like to take the readers through a deep dive into how complex a buildpipe can be.
+One of the larger open-source build pipelines is the rust programming language buildpipe,
+with lots of buildtargets, advanced conditions and bots for building just what is needed for different events.
```

Comment:
Add more details, eg. outline the sections of your essay.

Reply:
```suggestion
In this essay we would like to take the readers through a deep dive into how complex a buildpipe can be.
One of the larger open-source build pipelines is the rust programming language buildpipe,
with lots of buildtargets, advanced conditions and bots for building just what is needed for different events.

## Preliminary Outline
-  introduction
  - What is Rust
  - What is a buildpipe
  - What are GitHub Actions
- Parts of the Rust Build buildpipe
  - bots
     - [\@bors](https://bors.rust-lang.org/)
     - [\@rustbot](https://github.com/rust-lang/triagebot)
     - docs.rs
   - actions
      - docs
      - trying builds
      - releasing
- General Contribution process (linking bots and actions together)
```